unit-tests/code-snippets/get-nework.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { providers } from '@massalabs/wallet-provider';
+
+const networks = ['mainnet', 'testnet', 'buildnet', 'labnet'];
```

Comment:
This variable doesn't make sense if the test is removed. If you want to use it as code snippet, you should remove it and directly use the array in the expect.

Reply:
Thanks, I updated the example and added the code snippet like the generate-new-account test